logic/transaction_pool.js

```diff
@@ -14,17 +14,18 @@
 
 'use strict';
 
-var async = require('async');
-var config = require('../config.json');
-var constants = require('../helpers/constants.js');
-var jobsQueue = require('../helpers/jobs_queue.js');
-var transactionTypes = require('../helpers/transaction_types.js');
+const async = require('async');
+const config = require('../config.json');
+const constants = require('../helpers/constants.js');
+// eslint-disable-next-line  prefer-const
+let jobsQueue = require('../helpers/jobs_queue.js');
```

Comment:
`jobsQueue` doesn't get reassigned and could be const or do I get something wrong?

Reply:
in the tests, we rewire the module and set the values for stubbing, so if we declare it as const we can reassign. Here is the sample of how we set in tests `TransactionPool.__set__('jobsQueue', jobsQueue);`